templates/apiary_bucket_policy.json

```diff
@@ -18,6 +18,20 @@
                 "arn:aws:s3:::${bucket_name}/*"
             ]
         },
+%{if deny_iamroles != ""}
+        {
+            "Sid": "Local role deny permissions",
+            "Effect": "Deny",
+            "Principal": "*",
+            "Action": "s3:*",
+            "Resource": "arn:aws:s3:::${bucket_name}/*",
+            "Condition": {
+              "StringLike": {
+                "aws:PrincipalArn": [ "${deny_iamroles}" ]
+              }
+            }
+        },
+%{endif}
```

Comment:
What is the intent of this change?  For thse IAM roles, not allowing *any* actions (such as `List*`), seems overly punitive.  If people can't even see if their file was/wasn't created in the bucket correctly by listing the contents, it will just create more support incidents I would think.

Reply:
List still seems to works as that is a bucket operation instead of object, in this PR we are applying deny to bucket/*, which is only restricting access to objects.

Reply:
Ah, ok.  I missed that.  Sounds good.